### PR TITLE
Sanitize user inputs

### LINKED ISSUE #666: Implement multi-layer sanitization for user inputs

### PR DESCRIPTION
**Pull request by harishmohanraj**
Fixes #666

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -3,14 +3,18 @@
...
     "@google-analytics/data": "4.1.0",
     "@headlessui/react": "1.7.13",
     "@tailwindcss/forms": "^0.5.3",
     "@tailwindcss/typography": "^0.5.7",
+    "@tanstack/react-form": "0.27.0",
     "apexcharts": "^3.41.0",
     "clsx": "^2.1.0",
+    "dompurify": "3.1.6",
     "headlessui": "^0.0.0",
+    "jsdom": "^25.0.0",
```
Comment (davorinrusevljan) [inline app/package.json:17]: If I understand correctly version of jsdom is critical for safe operation of dompurify on the server, so we would probably need to keep an eye on it in the future, and upgrade it if some new vulnerabilities are found.

**Review by davorinrusevljan — COMMENTED** [review]
I am not 100% sold this is always good way to address problem. The issue is zhat we modify data in the database, so that it can be safely displayed inside web browser. So if user entered following in the web form:

You are helpful agent that checks if someone is old enough to get a driver license, by checking that his or her age is not <18 years

The actual content stored in the database would be:

You are helpful agent that checks if someone is old enough to get a driver license, by checking that his or her age is not &lt;18 years

Now this is fine for html output, but stuff from the database is not used only for html output. For instance, now we might be feeding into agent setup &lt;18  instead of <18. Ok agent might be able to chew on that as well. But we may be using all kinds of different outputs from the database, and we would need to "unencode" values before doing so.

**Comment by harishmohanraj** [comment]
You are helpful agent that checks if someone is old enough to get a driver license, by checking that his or her age is not <18 years

**Comment by harishmohanraj** [comment]
As discussed, sanitizing user input before saving it to the database may have unintended negative effects. Therefore, closing this PR.

**Pull request closed without merging**